Source/quests.cpp

```diff
@@ -445,10 +445,9 @@ bool ForceQuests()
 			int qy = quest.position.y;
 
 			for (int j = 0; j < 7; j++) {
-				if (qx + questxoff[j] == cursmx && qy + questyoff[j] == cursmy) {
+				if (qx + questxoff[j] == cursPosition.x && qy + questyoff[j] == cursPosition.y) {

```

Comment:
```suggestion
				if (Point { qx + questxoff[j], qy + questyoff[j] } == cursPosition) {
```

Reply:
Went with a different approach. Let me know if you are ok with it.